contributor-growth/drafts/contributor-growth-framework/1-Motivating.md

```diff
@@ -0,0 +1,107 @@
+You can even track things like who received what swag, ensuring you don't resend the same goodie twice.  
+
+1. Leverage "tasks" for anything that needs a follow up: Whether a blog post someone wanted to write or additional info you promised to provide. If it's an action item and you won't tackle it right away, create a task.
+2. Monitor reports on activity, engagement level, or community connection: Is engagement growing or dropping? Do you have a healthy distribution of users, contributors, and maintainers? How much does your community interact? Understanding these dynamics will help point at potential issues that you can actively address. For raw statistics, you can use CNCF DevStats which has a lot of data.
+3. Use tags: Tags can be based on interest, subprojects, or whatever makes sense for your project. Let's say you want to find contributors for your next observability feature on the roadmap. If used correctly, tags allow you to identify everyone who's expressed interest in that area.   
+
+These CRMs do lots of cool stuff. Savannah, for instance, allows you to see the impact your swag has on contributor productivity. 
+Another open source CNCF project is the [CHAOSS Project](https://chaoss.community/about/) which provides guidelines and processes for measuring the health of a community. It's not necessary to use CHAOSS or its principles for your project, but it is good to consider the overall health of your community and what you will use to measure that health as the community grows.  
+
+Tool | What it does
+---|---
+Chaoss | The Augur and Grimoire Lab projects can be integrated with the GitHub repositories for your project to view the commits and people who are making those commits over time.
+Savannah | Community CRM with GitHub, GitLab, Slack, Discourse, Discord, Stack Exchange integration
+Orbit | Community CRM with Discourse, GitHub, and Twitter, Slack (beta) integration.  1 
+
+
+## Metrics for Checking Contributor Growth
+
+A good starting point for finding contribution metrics about a CNCF project is on Devstats. Every CNCF project has a set of dashboards that display metrics from different sources, and the most relevant source for understanding contributor growth is the repository where commits are made for a project. At the time of this writing, all projects are hosted on GitHub and there is a dashboard that shows the number of contributors and contributions. Here is an example.
+
+## Contributor Recruitment Tactics  
+
+You can also engage in targeted efforts to recruit contributors. Tactics include new contributor workshops, contributor playground, AMAs/office hours/meet the maintainers, hackathons, and more. To learn more about this, please refer to the [recruiting playbook](https://github.com/cncf/sig-contributor-strategy/pull/63/files#diff-c3fcb95b44f62636424a6a0a1ed2f6598a11f8ca8011266d1e55fb2d80554804).  
```

Comment:
I think this may be a better URL for the playbook

https://github.com/cncf/sig-contributor-strategy/pull/63/files

The other one may end up being a bad link once more commits are pushed to that PR.

Reply:
Thanks!